wavelength-plugins/profile.js

```diff
@@ -499,6 +499,20 @@ exports.commands = {
 		"/nature delete - Removes your Profile Nature.",
 	],
 
+	'!lastactive': true,
+	checkactivity: 'lastactive',
+	lastactive: function (target, room, user) {
+		if (!target) target = user.userid;
+		if (target.length > 18) return this.errorReply("Usernames cannot exceed 18 characters.");
+		if (!this.runBroadcast()) return;
+		let targetUser = Users.get(target);
+		if (!Users(targetUser) || !targetUser.connected) return this.errorReply(`${target} is not online. Use /seen to find out how long ago they left.`);
+		if (targetUser.connected) {
+			this.sendReplyBox(`${WL.nameColor(targetUser, true, true)} was last active <strong>${Chat.toDurationString(Date.now() - targetUser.lastMessageTime)}</strong> saying <strong>"${targetUser.lastMessage}"</strong>.`);
```

Comment:
I'm pretty sure you forgot an "ago" after `</strong>`? Also, it's a really large potential security leak if the user's last-sent message is anything regarding sensitive information in the staff rooms.